SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -467,8 +470,14 @@ void USpatialNetDriver::CreateAndInitializeLoadBalancingClasses()
 
 	VirtualWorkerTranslator = MakeUnique<SpatialVirtualWorkerTranslator>(LoadBalanceStrategy, Connection->GetWorkerId());
 
-	LoadBalanceEnforcer =
-		MakeUnique<SpatialLoadBalanceEnforcer>(Connection->GetWorkerId(), StaticComponentView, VirtualWorkerTranslator.Get());
+	SpatialGDK::FSubView& LBSubView = Connection->GetCoordinator().CreateSubView(
+		SpatialConstants::LB_TAG_COMPONENT_ID, SpatialGDK::FSubView::NoFilter, SpatialGDK::FSubView::NoDispatcherCallbacks);
+
+	TUniqueFunction<void(SpatialGDK::EntityComponentUpdate AclUpdate)> AclUpdateSender = [&](SpatialGDK::EntityComponentUpdate AclUpdate) {
+		Connection->GetCoordinator().SendComponentUpdate(AclUpdate.EntityId, MoveTemp(AclUpdate.Update));
```

Comment:
Why do we need to call SendComponentUpdate on the coordinator directly and not just through the connection?

Reply:
This is a great question
It's because it also updates the main spatial view's state. However, then the updates are queued until the next flush, and I was hoping this would make it async. I'm going to look into whether this has to be the case. Thanks for pointing this out

Reply:
If this stays this way, please add a comment explaining it :)

Reply:
added a comment. The comment also explains the idea behind this, which it doesn't quite achieve yet, because short circuiting. A standalone worker running this would be a big improvement, even before full strategy worker.